README.adoc

```diff
@@ -85,7 +89,7 @@ for each Specification:
 | cxxhtml   | HTML outputs for C++ Specification
 | cxxpdf    | PDF outputs for C++ Specification
 |           |
-| env       | HTML and PDF outputs for Environment Specification
+| env       | HTML and PDF outputs for SPIR-V Environment Specification
 | envhtml   | HTML outputs for Environment Specification
 | envpdf    | PDF outputs for Environment Specification
```

Comment:
If we're going for 100% consistency might as well change these two as well.

Reply:
Can do. In the repository structure section above, abbreviated names of the specifications are used after the first mention of a specification, dropping "OpenCL" in subsequent mentions. I thought it might be preferable to do the same here, and drop "OpenCL SPIR-V" since it is clear what the specific environment specification is on these two lines. Given this explanation, do you still think it would be better to be unambiguous in all places, or is conciseness preferable?

Reply:
The current concise approach is fine, no action is required for this comment, I'll resolve it.